episodes/06-raster-intro.md

```diff
@@ -17,17 +17,21 @@ exercises: 30
 -  Visualize single/multi-band raster data.
 :::
 
-
-:::callout
```

Comment:
This callout is pretty big and it describes various aspects of this episode - from introducing the programs to how to get help. I think it would be better to focus the callout on the latter topic?